testsuite/k8s-conformance-test-suite.sh

```diff
@@ -59,6 +59,11 @@ while IFS= read -r arc_platform_version || [ -n "$arc_platform_version" ]; do
 
     sonobuoyResults=$(sonobuoy retrieve)
     sonobuoy results $sonobuoyResults
+
+    mkdir testResult
+    python arc-k8s-platform/remove-secrets.py $sonobuoyResults testResult
```

Comment:
So now this will introduce a dependency on python. We should document this in the pre-requisites section. When we implement the new proposal, this dependency should go away, so i guess its okay to keep it for now

Reply:
done